docs/using.rst

```diff
@@ -28,6 +28,32 @@ Starting the Vagrant Box
 
    **Note:** On Fedora and CentOS you may need to specify which virtualization provider to use.  For example, to use VirtualBox, the command would be ``vagrant up --provider virtualbox``
 
+Using the box from the Host
+===========================
+
+Many users may wish to use the ADB from their Host so it can seamlessly interact with their files, preferred development tools, etc.
+
+Today, the only support is for the exposure of the docker daemon port.  The docker daemon is TLS protected, so in addition to exposing the port you must configure the docker command on the host to use TLS and the right port.  This can be done via environment variables.
+
+1. Install the ``vagrant-adbinfo`` plugin in order to get easy access to the correct environment variables and the TLS certificates.
+
+   ``vagrant plugin install vagrant-adbinfo``
+
+More information about the vagrant-adbinfo plugin is `available in the source repository <https://github.com/bexelbie/vagrant-adbinfo>`_
+
+2. Get the correct environment variables and TLS certificates from the plugin.  The example below shows the command and the output::
+
+    $ vagrant adbinfo
+    Set the following environment variables to enable access to the
+    docker daemon running inside of the vagrant virtual machine:
+    
+    export DOCKER_HOST=tcp://172.13.14.1:5555
+    export DOCKER_CERT_PATH=/home/bexelbie/Repositories/vagrant-adbinfo/.vagrant/machines/default/virtualbox/.docker
+    export DOCKER_TLS_VERIFY=1
+    export DOCKER_MACHINE_NAME="90d3e96"
+
+Setting these environment variables allows commands, such as docker to access the docker daemon.  There is no need to pass any special options to the ``docker`` command.
+
```

Comment:
I think we should document how an user can add this information to environments of Windows , OS X and GNU/Linux.